kong/dao/db/postgres.lua

```diff
@@ -631,4 +631,15 @@ function _M:reachable()
   return true
 end
 
+function _M:refresh()
+  -- This is a dummy function to provide compatibility.
+  -- Refresh only applies to Cassandra.
+  local ok, err = true, true
+  if not ok then
+    return nil, Errors.db(err)
+  end
+
+  return true
```

Comment:
minor suggestion: no need to keep the "error handling template" in this Postgres implementation. In similar situations we just keep it simple, with the body keeping only `return true` and the comment.